recipes-bsp/imx-oei/imx-oei_1.0.0.bb

```diff
@@ -0,0 +1,57 @@
+SUMMARY = "i.MX Optional Execution Image"
+
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=59530bdf33659b29e73d4adb9f9f6552"
+
+INHIBIT_DEFAULT_DEPS = "1"
+DEPENDS = "gcc-arm-none-eabi-native"
```

Comment:
The recipe does depend on `meta-arm`, should it be in `dynamic-layers`?

Reply:
Good catch, but this must be fixed in master first.

Reply:
Is this about just moving the imx-oei recipe to `meta-freescale/dynamic-layers/meta-arm`? If so, I can do it on master and then cherry-pick the commit here, otherwise we could proceed with the PR and solve it later, whatever you guys think is the best.

Reply:
You should use the directory with the name of the collection provided by the layer. 
so please prepare a pull request for master and then we do the backport automatically using the github actions

Reply:
https://github.com/Freescale/meta-freescale/pull/1910 please let me know if this is correct